src/v0/destinations/salesforce/utils.js

```diff
@@ -33,16 +33,21 @@ const salesforceResponseHandler = (destResponse, sourceMessage, authKey, authori
     const matchErrorCode = (errorCode) =>
```

Comment:
To make the code readable, the above `if` condition can be moved to calling class. There's no point of having only if without else in this function

Reply:
this can be part of different PR